### ISSUE TITLE
Maybe it was already posted somewhere, but is there blog/news somewhere about new releases of typeorm already?

### ISSUE THREAD

**Issue by idchlife** [typeorm/typeorm#1252]
Kinda .save was out of the blue for me.

Or are there release notes planned? I cannot see them in releases list.

Will be very helpful!

**Comment by pleerock** [comment]
Hey, lot of things has changed in a last year. The most recent release is `0.1.x` and you can read [changelog here](http://typeorm.io/#/changelog). We are actively working on `0.2.0` you can read about changes [here](https://github.com/typeorm/typeorm/blob/next/CHANGELOG.md).

**Comment by idchlife** [comment]
@pleerock thank you!

**Comment by pleerock** [comment]
did you come back from java?)

**Comment by idchlife** [comment]
@pleerock ?

**Comment by pleerock** [comment]
I saw your reddit/stackoverflow questions about decision regarding typescript or java and I didn't see you here for a while so I thought you were in java zone all this time

**Comment by idchlife** [comment]
@pleerock actually I tried Java at the time I was "absent", you are right :D
But typeorm;routing-controllers;typedi is always here, I have like 4 projects using them, one of which is for working with payments in production. I'm not always updating libraries, though...

Project I was asking about is way too complicated so I started digging in different directions looking for as much protection from the language side as I could. It's still in planning stage so there will be more digging. But making nice working payment service with typescript gave TS more weight when time to choose will come.

There is also C# with ASP.NET... it's nice. Very nice. And it works ok with linux and docker nowadays.

As much as I love transpiled to js languages and new static typed languages, I still have to pick one with stable production history and mature ecosystem. So no luck for elm, elixir and crystal or even dart.

TypeScript on the other hand has some kind of dawn right now. Your libraries is one huge part of it.
There still no predefined directory structure for new project and there is still "bootstrap" functions for firing up express with lines of code which are not fully understandable for newcomers without comments, but... Improvements and ideas will come, I think.
